client_device_provisioning/fleet_provisoning/main.py

```diff
@@ -0,0 +1,42 @@
+import sys, getopt
+from config import Config
+from handler import Handler
+
+# Provided callback for provisioning method feedback.
+def callback(payload):
+    print(payload)
+
+def run_provisioning(argv):
+    config = Config()
+    opts, args = getopt.getopt(argv,"he:p:s:n:t:",["endPoint=", "templateName=","serialNumber=","thingName=","topic="])
```

Comment:
Recommend using `click` as a library for making really nice CLIs. Also allows help messages etc.